### ISSUE TITLE
Run-AlValidation doesnt accept Third-Party Runtime-App

### ISSUE THREAD

**Issue by maxgkm** [microsoft/navcontainerhelper#2783]
**Describe the issue**
Hello,

i tried to use Run-AlValidation to validate my extension and make sure its ready to be uploaded into the AppSource.
My App has a dependency for which i have the Runtime-App from a third party. This App can be installed and used without a problem on a regular container. If i specify this Runtime-App to be installed while the AlValidation it crashes with the below specified error.

**Scripts used to create container and cause the issue**
```
Run-AlValidation `
    -apps "<path to my app>" `
    -affixes "<my affix>" `
    -countries "de,at,ch" `
    -installApps "<Path to third party Runtime App>"
```

**Full output of scripts**
```
Cannot unpack app <Third party Runtime App File Name>, it might be a runtime package.
In C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\4.0.6\AppHandling\Run-AlValidation.ps1:188 Zeichen:9
+         throw "Cannot unpack app $([System.IO.Path]::GetFileName($app ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Cannot unpack a...untime package.:String) [], RuntimeException
    + FullyQualifiedErrorId : Cannot unpack app <Third party Runtime App File Name>, it might be a runtime package.
```

**Screenshots**
/

**Additional context**
At first i used BcContainerHelper 3.0.6 which resulted in the error.
Now i updated to 4.0.6 and it still crashes.

**Comment by freddydk** [comment]
If you specify the version you want to validate against (using -validateVersion) - or specify -validateCurrent, -validateNextMinor or -validateNextMajor - then it should work.

It fails in the code trying to find out which version to validate for.
Will fix this by ignore the runtimepackages when determining highest minimum version.

**Comment by freddydk** [comment]
Shipped in 5.0.0
